src/state_machine/coordinator/mod.rs

```diff
@@ -90,6 +91,9 @@ pub enum Error {
     /// Supplied party polynomial contained duplicate party IDs
     #[error("Supplied party polynomials contained a duplicate party ID")]
     DuplicatePartyId,
+    /// An error during elliptic curve operations
+    #[error("An elliptic curve error occurred: {0}")]
+    EllipticCurveError(String),
```

Comment:
Same comment about just carrying around the actual error in this variant. Why not do that here?